pims/tests/test_common.py

```diff
@@ -570,13 +571,20 @@ def setUp(self):
         self.v = self.klass(self.filename, **self.kwargs)
         self.expected_shape = shape
         self.expected_len = 5
+        with zipfile.ZipFile('test.zip', 'w') as archive:
```

Comment:
No, I don't trust that this won't run in a directory where there is an existing user file called 'test.zip'